CraftTweaker2-MC1120-Mod-JEI/src/main/java/crafttweaker/mods/jei/JEI.java

```diff
@@ -30,43 +29,37 @@
 public class JEI {
     
     public static List<IAction> LATE_HIDING = new LinkedList<>();
+    public static List<IAction> DESCRIPTIONS = new LinkedList<>();
```

Comment:
DESCRIPTION needs to be a separate list, as this can't be called during the OnLoadCompleteEvent
At least I couldn't get it to work that way...
But I can condense LATE_Hiding and LATE_Addition to LATE_ACTIONS